stdpopsim/cli.py

```diff
@@ -436,11 +439,22 @@ def run_simulation(args):
         if not args.quiet:
             write_simulation_summary(engine=engine, model=model, contig=contig,
                                      samples=samples, seed=args.seed)
+        if not qc_complete:
+            warnings.warn(
+                    f"{model.id} has not been QCed. Use at your own risk! "
+                    "Demographic models that have not undergone stdpopsim's "
+                    "Quality Control procedure may contain implementation "
+                    "errors, leading to differences between simulations "
+                    "and the model described in the original publication. "
+                    "More information about the QC process can be found in "
+                    "the developer documentation. "
+                    "https://stdpopsim.readthedocs.io/en/latest/development.html"
+                    "#demographic-model-review-process")
         ts = engine.simulate(**kwargs)
         summarise_usage()
         if ts is not None:
             write_output(ts, args)
-        if not args.quiet:
+        if qc_complete and not args.quiet:
```

Comment:
Not sure why you're tying QC status to citation output here - we'll still have citations for other stuff that has been QC'd like the species definition. Are you thinking that "you shouldn't be using this in a publication anyway", so there's no point in writing out the citation info? Or maybe the warning gets lost in the output with all the cites?

A comment would be helpful I think.

Reply:
Immediately following the warning, the citation information would print, which starts with `If you use this simulation in published work, please cite:`. Which seems a little odd, given that we just warned against using the model. So, yeah, I kinda figured the user shouldn't be using non-QCed models in a publication. Happy to add a comment in the code to this effect.